### DISCUSSION TITLE
非常非常好看的主题

### DISCUSSION THREAD

**Discussion by kuuhaku-w** [YunYouJun/hexo-theme-starry#140] [Show 分享展示]
[熊之记事本](https://kuuhaku.top)
---
blog: 熊之记事本
url: https://kuuhaku.top
avatar: https://cdn.jsdelivr.net/gh/kuuhaku-w/blogimg/intro/avatar_ai.png
desc: 永远是深夜该有多好
---
---
云游君的主题真的超好看啊！非常喜欢！而且使用指南也写得很详细，从来没见过这么贴心的使用说明以至于在内心不停感叹：“果然这才是真正的使用说明吧！”
另外关于二次元之间会互相吸引这件事。。。果然是真的？qvq

**Comment by YunYouJun** [comment]
哈哈哈，谢谢！自定义的也很好看！
（下次会加入示例站点的）

**Reply by YunYouJun** [reply]
已加入示例站点。

**Reply by kuuhaku-w** [reply]
好耶～
